### DISCUSSION TITLE
Display counter for Exarch & Eater spawns

### DISCUSSION THREAD

**Discussion by GrimDawn-max** [Lailloken/Exile-UI#205] [Ideas & Feedback]
Whenever you enter a map and spawn an Exarch or Eater mob encounter the progress counter for these goes up by one. Unfortunately you have to press the G key to check what the counter is (or access the map device and check that way). Would it be possible to have a display overlay that shows the counters for each instead so don't have to access the map atlas or map device to see them ? This would shave a few seconds off an activity I'm currently pursuing (I don't think there is an in game command to do so but even that would take time so display would be nicer).

**Comment by Lailloken** [comment]
I just checked, and unfortunately the client-log doesn't distinguish between normal and influenced maps, so there's no way to trace or calculate the current state of the counter.

**Reply by GrimDawn-max** [reply]
Thanks for checking. It was a long shot but thought I'd ask
